test/wsl.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -x
+
+./bin/vm \
+    -url="stdio:/home/ubuntu/gvisor-tap-vsock/bin/gvproxy-windows.exe?listen-stdio=accept&debug=true" \
```

Comment:
can this work with `$HOME` ? There is no `ubuntu` user on Fedora ;-)

Reply:
Used `pwd` to make it a bit more flexible.